instrumentation/spring/spring-security-config-6.0/javaagent/README.md

```diff
@@ -0,0 +1,14 @@
+# OpenTelemetry Javaagent Instrumentation: Spring Security Config
+
+Javaagent automatic instrumentation to capture `enduser.*` semantic attributes
+from Spring Security `Authentication` objects.
+
+## Settings
+
+| Property                                                                      | Type    | Default  | Description                                                                                             |
+|-------------------------------------------------------------------------------|---------|----------|---------------------------------------------------------------------------------------------------------|
+| `otel.instrumentation.common.enduser.id.enabled`                              | Boolean | `false`  | Whether to capture `enduser.id` semantic attribute.                                                     |
```

Comment:
Common configuration propertied should be documented in https://opentelemetry.io/docs/instrumentation/java/automatic/agent-config/#common-instrumentation-configuration I'm not sure whether it makes sense to also document them here

Reply:
I updated the readme to point to that location for the common properties (assuming https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/9783 will be done soon)